### ISSUE TITLE
Fix cartesian method

### ISSUE THREAD

**Issue by damjanh** [BlakeGuilloud/ganon#430]
There is currently a method called `cartesian` that lives in `/lib/cartesian.js`.
It is incomplete and needs to be fixed!

The method should return an array constructed with the rules for the cartesian product. Insure that the only valid parameters are arrays.

### Acceptance Criteria:
1. Running `yarn test cartesian` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

**Comment by rgehan** [comment]
I'll have a look at it :)

**Comment by FranceJ** [comment]
@BlakeGuilloud looks like @rgehan wants to work on this. Can you apply the tags?

**Comment by rgehan** [comment]
@BlakeGuilloud Method is fixed, I added another issue (#433), but this one needs to be merged first :)
